.github/workflows/test.yaml

```diff
@@ -0,0 +1,31 @@
+name: 'CI'
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+
+jobs:
+  run-tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        pythonversion: ['2.7', '3.5', '3.6', '3.8', '3.8', '3.9', 'pypy-2.7', 'pypy-3.6']
+    steps:
+      - uses: actions/checkout@v2
+      - name: set up python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.pythonversion }}
+      - name: install dependencies
+        run: |
+          python -m pip install six requests flake8
+          python -m pip install -r requirements-tests.txt
+          python -m pip install -e .
```

Comment:
Won't `python -m pip install -e .` also be installing the requirements from `setup.py` and so installing `six` and `requests` separate above would be redundant?

Reply:
probably. I was just translating from the travis config, which copied that from the circle config, which got it from gods know where